### PR TITLE
Fix TLS issues

### PR DESCRIPTION
**Pull request by CyberHoward**
Fixes the TLS related issues that we've ran into for https URLs

### CODE REVIEW

File: cw-orch-daemon/Cargo.toml [hunk 1]
```diff
@@ -45,7 +45,7 @@ bech32 = { version = "0.11.0", default-features = false, features = ["alloc"] }
 hex = { version = "0.4.3" }
 ripemd = { version = "0.1.3" }
 tokio = { workspace = true, features = ["full"] }
-tonic = { workspace = true, features = ["tls", "tls-roots"] }
+tonic = { workspace = true, features = ["tls"] }
```
Comment (Kayanski) [inline cw-orch-daemon/Cargo.toml:48]: Reallt removing the tls roots here ?

File: cw-orch-daemon/src/channel.rs [hunk 2]
```diff
@@ -2,7 +2,8 @@ use cosmrs::proto::cosmos::base::tendermint::v1beta1::{
     service_client::ServiceClient, GetNodeInfoRequest,
 };
 use cw_orch_core::{environment::ChainInfoOwned, log::connectivity_target};
-use tonic::transport::{Channel, ClientTlsConfig};
+use http::Uri;
```
Comment (Kayanski) [inline cw-orch-daemon/src/channel.rs:5]: Have you tried this new code with `http://` prefixed URIs ?

**Review by Kayanski — APPROVED** [review]

File: cw-orch-daemon/Cargo.toml (hunk 1 above)
Comment (CyberHoward) [reply cw-orch-daemon/Cargo.toml:48]: It works without so not sure why we should enable it.

File: cw-orch-daemon/src/channel.rs (hunk 2 above)
Comment (CyberHoward) [reply cw-orch-daemon/src/channel.rs:5]: Yes, see the tests in `query_only.rs`. There's a http and https test there.

**Pull request merged**